src/eviction.cpp

```diff
@@ -282,6 +282,14 @@ void EvictionManagerImpl::UpdateLoadedBloomFilter(NodeId id, bool bloom_filter_l
     }
 }
 
+void EvictionManagerImpl::UpdateRelayTxs(NodeId id, bool relay_txs)
```

Comment:
Since this can never become false again, I think we don't need the bool?

That has the advantage of communicating that it can't be unset.